packages/scaffolding-cli/src/__tests__/domain/workers/integration_testflow.test.ts

```diff
@@ -33,20 +33,20 @@ describe("mainWorker class", () => {
     const currentDir = process.cwd()
     let tempDir: string
 
-    beforeEach(() => {
+    beforeEach(async () => {
```

Comment:
@dnitsch is this working for you? I had issues doing this asynchronously when running from root. The snapshots weren't writing correctly. The generator function would still be running and then the snapshot would have files that were from another test.

Reply:
yes it is actually. I somehow still remember that it's not meant to, but it is - even in their docs they say beforeEach work the same way as tests...